packages/ui-list/src/List/ListItem/props.ts

```diff
@@ -40,10 +40,26 @@ import type {
 
 type ListItemOwnProps = {
   children: React.ReactNode | ((...args: any[]) => React.ReactNode)
```

Comment:
```suggestion
  children: React.ReactNode | (() => React.ReactNode)
```
It doesn't receive any parameters.